src/src/components/Toast/toast.module.css

```diff
@@ -1,67 +1,50 @@
-.container_toast {
-  display: flex;
-  background-color: rgba(190, 30, 45, 0.8);
-  width: 284px;
-  cursor: default;
-  position: fixed;
-  left: 50%; /* Position the element at the horizontal center */
-  transform: translateX(-50%);
-  bottom: 1rem;
-  z-index: 10000;
-  box-sizing: border-box;
-  display: inline-flex;
-  align-items: center;
-  padding: 0.5rem;
-  border-radius: 5px;
-  justify-content: center;
-}
+/* toast.module.css */
 
-/* Toast Close Button */
-.toast_close {
-  display: flex;
-  cursor: pointer;
-  color: #fff;
-  justify-content: center;
-  align-items: center;
-  padding: 0 0 0 0;
-  margin: 0 0 0 20px;
-  line-height: 16px;
-  border-radius: 5px;
-}
-
-.toast_close:hover {
-  background-color: #920916;
+.container_toast {
+  background-color: rgba(190, 30, 45, 0.8); /* Red translucent background */
+  color: white; /* Text color */
+  max-width: 350px; /* Maximum width */
+  min-width: 248px; /* Default width */
+  height: auto; /* This ensures that the height changes depending on content */
+  min-height: 100px; /* Default height */
+  padding: 10px; /* Add some space around the text and button */
+  box-sizing: border-box; /* Ensures padding is included in the total width/height */
+  border-radius: 8px; /* Optional: To give the toast rounded corners */
+  overflow-wrap: break-word; /* Ensure words break properly if they exceed the container's width */
+  display: flex; /* To align the text and close icon horizontally */
+  align-items: center; /* Center-align items vertically */
+  justify-content: space-between; /* Space out the text and close icon */
+  margin-bottom: 10px; /* Optional: Space between multiple toasts */
+  height: auto;
+  max-height: none;
+  word-wrap: break-word;
+  position: relative; /* This allows absolute positioning for child elements like the close button */
 }
```

Comment:
this looks a lot like chatgpt. is it? I think its ok to use but im worried about large amounts of code/css is pasted in rather than changing just the bits that needs changing

Reply:
It's multiple chatGPTs

Reply:
😬